cms/static/js/views/pages/container.js

```diff
@@ -25,7 +25,7 @@ define(['jquery', 'underscore', 'backbone', 'gettext', 'js/views/pages/base_page
 
             options: {
                 collapsedClass: 'is-collapsed',
-                canEdit: true // If not specified, assume user has permission to make changes
+                canEdit: true// If not specified, assume user has permission to make changes
```

Comment:
Nit: Was the removal of the space here necessary for linting?

Reply:
Pretty sure it was because at one point I had made changes here. I'll fix